lib/document-to-html.js

```diff
@@ -224,12 +224,21 @@ function DocumentConversion(options, comments) {
         return idPrefix + suffix;
     }
 
-    var defaultTablePath = htmlPaths.elements([
-        htmlPaths.element("table", {style: "border-collapse: collapse;"}, {fresh: true})
-    ]);
+    function getDefaultTable(element) {
+        var attributes = {
+            style: "border-collapse: collapse;"
+
+        };
+        if (element.width && element.widthUnit === "dxa") {
```

Comment:
How about other units?

Reply:
Actually, maybe it makes sense to support percent for the table width as well. It doesn't hurt to add it here anyway